ioc_module.js

```diff
@@ -108,7 +108,7 @@ function registerServices(container) {
 
   container
     .register('ProcessModelService', ProcessModelService)
-    .dependencies('ProcessDefinitionRepository', 'IamService', 'BpmnModelParser');
+    .dependencies('CorrelationService', 'ProcessDefinitionRepository', 'IamService', 'BpmnModelParser');
```

Comment:
```suggestion
    .dependencies('CorrelationRepository', 'ProcessDefinitionRepository', 'IamService', 'BpmnModelParser');
```

Always reference the repositories here, **not** the services. 
Referencing services in other services could easily lead to circular dependencies.